frontend/src/components/Candidate-Decider/CandidateDecider.tsx

```diff
@@ -68,7 +79,13 @@ const CandidateDecider: React.FC<CandidateDeciderProps> = ({ uuid }) => {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [currentCandidate, instance.candidates, reviews]);
 
-  const next = () => {
+  const next = (force = false) => {
+    if (!force && !isSaved && !isBypassingModal) {
```

Comment:
Now that we're not using `navigationDirection` anymore for the text, it feels like `prev`, `next`, and `handleCandidateChange` can all be condensed into just one function. Can you try deleting `prev` and `next` and just using `handleCandiateChange`? (you might have to check if `candidate` number is within `0...instance.candidates.length - 1` but otherwise the looks the same to me)

and then we can get rid of `navigationDirection `!